tools/flang2/flang2exe/ll_structure.cpp

```diff
@@ -394,10 +394,12 @@ compute_ir_feature_vector(LLVMModuleRef module, enum LL_IRVersion vers)
     module->ir.dwarf_version = LL_DWARF_Version_2;
   } else if (XBIT(120, 0x4000)) {
     module->ir.dwarf_version = LL_DWARF_Version_3;
-  } else if (true) { // FIXME - need a new bit
+  } else if (XBIT(120, 0x1000000)) {
     module->ir.dwarf_version = LL_DWARF_Version_4;
-  } else {
+  } else if (XBIT(120, 0x2000000)) {
     module->ir.dwarf_version = LL_DWARF_Version_5;
+  } else if (true) { // FIXME - need a new bit
```

Comment:
Can you make this a simple `else` clause? And add a comment that DWARF 4 is the default? (This will fix the FIXME. Yay.)

Reply:
Thanks a lot for your comment. I shall be updating with your suggestion.